automatic/qbittorrent/tools/chocolateyinstall.ps1

```diff
@@ -7,7 +12,7 @@ $packageArgs = @{
   fileType       = 'exe'
   softwareName   = 'qBittorrent*'
   file           = "$toolsDir\qbittorrent_4.4.5_setup.exe"
-  file64         = "$toolsDir\qbittorrent_4.4.5_x64_setup.exe"
+  file64         = "$toolsDir\qbittorrent_4.5.0_x64_setup.exe"
```

Comment:
The automatic update script should do this so you shouldn't need to change this here.